source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,65 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<% format_sassdoc_data(data).each do |heading, group| %>
+
+## <%= group_heading(heading) %>
+
+<% group.each do |doc| %>
+
+### <%= doc_heading(doc) %>
+<span class="govuk-secondary-text">(<%= doc.context.type %>)</span>
```

Comment:
As far as I can tell, this class doesn't seem to exist?

Reply:
I think I accidentally committed this, oops thanks for catching.